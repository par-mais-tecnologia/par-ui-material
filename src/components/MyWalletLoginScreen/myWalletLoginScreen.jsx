import React from 'react'
import Input from '@material-ui/core/Input'
import Button from '@material-ui/core/Button'

import '../Core/styles/index.css'

const MyWalletLoginScreen = (props) => {
  return (
    <div className='flex z-2 justify-center w-100 h-100 justify-center items-center'>
      <div className='white pv4 pl4 mr5 dn db-ns'>
        <div className='brandon-regular lh-copy'>
          <p className='f4 mb2 rounded-elegance'>#ClienteMais</p>
          <p className='f3 ttu mt2 mb1'>Trabalhamos <br /> <span className='brandon-bold'>para você <br /></span>e para o
            seu dinheiro</p>
          <p className='f6 ttu mt2 lh-copy'>Seus investimentos e vida financeira <br /> aos cuidados de especialistas</p>
        </div>
        <p className='roboto-bold f6 b'>Conheça a Par Mais</p>
      </div>
      <div className='bl-ns b--white-20 white roboto-medium pl5-ns pr5-ns mt4-ns'>
        <p className='f2 mb4 mt3 rounded-elegance dn-ns'>#ClienteMais</p>
        <p className='roboto-light f4'>Acesse sua conta</p>
        <div>
          <Input
            placeholder='E-mail'
            fullWidth
            className='mb3'
          />
          <br />
          <Input
            placeholder='Senha'
            fullWidth
          />
          <p className='f7 f6-ns roboto-bold b mt4 mb4 lh-copy'>
              Esqueceu sua senha?<br />
              Clique para receber uma nova em seu e-mail
          </p>
          <div className='flex justify-end'>
            <Button variant='contained' color='secondary'>
                Continuar
            </Button>
          </div>
        </div>
      </div>
    </div>
  )
}

export default MyWalletLoginScreen
