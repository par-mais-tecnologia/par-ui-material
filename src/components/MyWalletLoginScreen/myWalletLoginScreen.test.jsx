import React from 'react'
import { configure, shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

import BaseInitialScreen from '../BaseInitialScreen/baseInitialScreen'
import MyWalletLoginScreen from './myWalletLoginScreen'

configure({ adapter: new Adapter() })

const getShallow = () => (
  <BaseInitialScreen
    imageSrc={'https://wallhalla.com/thumbs/preview/l/La748MeuPdY.jpg'}
    middleBoxFullScreen={false}
    middleBoxColor='#2b4e99'>
    <MyWalletLoginScreen />
  </BaseInitialScreen>
)

describe('myWalletLoginScreen', () => {
  const wrapper = shallow(getShallow())

  it('shallow MyWalletLoginScreen', () => {
    expect(wrapper).toBeTruthy()
  })

  it('shallow MyWalletLoginScreen match Snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
