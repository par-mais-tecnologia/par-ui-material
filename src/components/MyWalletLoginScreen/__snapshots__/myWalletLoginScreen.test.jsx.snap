// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`myWalletLoginScreen shallow MyWalletLoginScreen match Snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <BaseInitialScreen
    imageSrc="https://wallhalla.com/thumbs/preview/l/La748MeuPdY.jpg"
    middleBoxColor="#2b4e99"
    middleBoxFullScreen={false}
    middleBoxFullScreenMobile={false}
  >
    <MyWalletLoginScreen />
  </BaseInitialScreen>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="flex middleBox flex ab row justify-center items-center flex-wrap  w-auto-ns h-auto-ns absolute"
        style={
          Object {
            "background": "inherit",
            "height": "",
            "overflow": "hidden",
            "width": "",
          }
        }
      >
        <div
          className="glassColor ma0-ns z-5 pa4 w-100 h-100"
          style={
            Object {
              "backgroundColor": "rgba(43, 78, 153, 0.6)",
            }
          }
        >
          <MyWalletLoginScreen />
        </div>
      </div>,
      "className": "flex img-bg justify-center absolute items-center w-100 h-100 cover",
      "style": Object {
        "backgroundAttachment": "fixed",
        "backgroundImage": "url(https://wallhalla.com/thumbs/preview/l/La748MeuPdY.jpg)",
        "backgroundRepeat": "no-repeat",
      },
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="glassColor ma0-ns z-5 pa4 w-100 h-100"
          style={
            Object {
              "backgroundColor": "rgba(43, 78, 153, 0.6)",
            }
          }
        >
          <MyWalletLoginScreen />
        </div>,
        "className": "flex middleBox flex ab row justify-center items-center flex-wrap  w-auto-ns h-auto-ns absolute",
        "style": Object {
          "background": "inherit",
          "height": "",
          "overflow": "hidden",
          "width": "",
        },
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <MyWalletLoginScreen />,
          "className": "glassColor ma0-ns z-5 pa4 w-100 h-100",
          "style": Object {
            "backgroundColor": "rgba(43, 78, 153, 0.6)",
          },
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": "div",
      },
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="flex middleBox flex ab row justify-center items-center flex-wrap  w-auto-ns h-auto-ns absolute"
          style={
            Object {
              "background": "inherit",
              "height": "",
              "overflow": "hidden",
              "width": "",
            }
          }
        >
          <div
            className="glassColor ma0-ns z-5 pa4 w-100 h-100"
            style={
              Object {
                "backgroundColor": "rgba(43, 78, 153, 0.6)",
              }
            }
          >
            <MyWalletLoginScreen />
          </div>
        </div>,
        "className": "flex img-bg justify-center absolute items-center w-100 h-100 cover",
        "style": Object {
          "backgroundAttachment": "fixed",
          "backgroundImage": "url(https://wallhalla.com/thumbs/preview/l/La748MeuPdY.jpg)",
          "backgroundRepeat": "no-repeat",
        },
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="glassColor ma0-ns z-5 pa4 w-100 h-100"
            style={
              Object {
                "backgroundColor": "rgba(43, 78, 153, 0.6)",
              }
            }
          >
            <MyWalletLoginScreen />
          </div>,
          "className": "flex middleBox flex ab row justify-center items-center flex-wrap  w-auto-ns h-auto-ns absolute",
          "style": Object {
            "background": "inherit",
            "height": "",
            "overflow": "hidden",
            "width": "",
          },
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <MyWalletLoginScreen />,
            "className": "glassColor ma0-ns z-5 pa4 w-100 h-100",
            "style": Object {
              "backgroundColor": "rgba(43, 78, 153, 0.6)",
            },
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": "div",
        },
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
